templates/product_simulator.php

```diff
@@ -65,6 +65,7 @@ function loadSimulatorPagantis()
     {
         if(typeof pgSDK == 'undefined')
         {
+            console.warn(typeof pgSDK)
```

Comment:
remove comments

Reply:
The file is not used anymore, so I will remove the file completely